test/e2e/e2e.go

```diff
@@ -50,9 +52,16 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	framework.Logf("checking config")
 
-	c, err := framework.LoadClientset()
+	// We don't want WaitForPodsRunningReady and pod logging to be throttled.
+	// Therefore we set a fake rate limiter which never throttles.
+	config, err := framework.LoadConfig()
 	if err != nil {
-		framework.Failf("Error loading client: %v", err)
+		framework.Failf("error creating client config: %v", err)
+	}
+	config.RateLimiter = flowcontrol.NewFakeAlwaysRateLimiter()
+	c, err := clientset.NewForConfig(config)
+	if err != nil {
+		framework.Failf("error creating client: %v", err)
```

Comment:
Test run *without* this commit:
https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/blue/organizations/jenkins/pmem-csi/detail/version-skew-2/22/pipeline/

Test run *with* it:
https://cloudnative-k8sci.southcentralus.cloudapp.azure.com/blue/organizations/jenkins/pmem-csi/detail/version-skew-2/23/pipeline